README.md

```diff
@@ -24,7 +24,6 @@ Install tools in your dev environment:
 - [docker](https://www.docker.com/)
 - [kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#install_kubectl)
 - [helm](https://helm.sh/)
-- [envsubst](https://linux.die.net/man/1/envsubst) (Optional)
```

Comment:
Do we still use envsubst? If so, we should remain envsubst.
https://github.com/googleforgames/space-agon/blob/main/scripts/setup-skaffold.sh#L46-L47

Reply:
Thanks. I missed that.